js/common/lib/tensor.ts

```diff
@@ -234,5 +234,71 @@ export interface TensorConstructor {
   // #endregion
 }
 
+export interface ImageFromTensorOptions {
+  format?: 'RGB'|'RGBA'|'BGR'|'RBG';
+  tensorLayout?: 'NHWC'|'NCHW';
+  height?: number;
+  width?: number;
+  norm?: {
+    normBias?: number;  // Todo add support - |[number,number,number]|[number,number,number,number];
+    normMean?: number;
+  };  // Todo add support - |[number,number,number]|[number,number,number,number];
+}
+export interface TensorFromImageOptions {
+  bitmapFormat?: 'RGB'|'RGBA'|'BGR'|'RBG';  // used only in the case of image bitmap is used for
+  tensorFormat?: 'RGB'|'RGBA'|'BGR'|'RBG';
+  dataType?: 'float32'|'uint8';
+  tensorLayout?: 'NHWC'|'NCHW';
+  height?: number;
+  width?: number;
+  tensorHeight?: number;
+  tensorWidth?: number;
+  norm?: {
```

Comment:
is `normalize` better? @guschmue 

I think this may worth to mention. I am ok with both

Reply:
either way is fine. tensorHeight and tensorWidth sounds a little odd: images have H/W, tensors have shapes.